packages/rln/src/contract/constants.ts

```diff
@@ -1,8 +1,9 @@
 import { RLN_ABI } from "./abi.js";
 
 export const SEPOLIA_CONTRACT = {
-  chainId: 11155111,
-  address: "0xCB33Aa5B38d79E3D9Fa8B10afF38AA201399a7e3",
+  chainId: 59141,
+  // Implementation contract: 0xde2260ca49300357d5af4153cda0d18f7b3ea9b3
```

Comment:
is `0xde2260ca49300357d5af4153cda0d18f7b3ea9b3` used in any way? if no - let's remove it OR document somewhere (e.g `README.md`)